From b0d95e79b7fa298a34660bc1973994283080037f Mon Sep 17 00:00:00 2001
From: Johannes Maibaum <jmaibaum@gmail.com>
Date: Thu, 21 Mar 2024 09:30:12 +0100
Subject: [PATCH] Add <launchable> tag to appdata

Flathub requires this to be present in the appstream metadata. Otherwise
builds are rejected from flathub.

See: https://buildbot.flathub.org/#/builders/6/builds/108488
---
 data/soundconverter.appdata.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/soundconverter.appdata.xml b/data/soundconverter.appdata.xml
index da85be9..3cf3342 100644
--- a/data/soundconverter.appdata.xml
+++ b/data/soundconverter.appdata.xml
@@ -28,6 +28,7 @@ SoundConverter is a simple to use, and very fast audio converter. It can convert
   <url type="translate">https://translations.launchpad.net/soundconverter</url>
 
   <content_rating type="oars-1.1" />
+  <launchable type="desktop-id">soundconverter.desktop</launchable>
 
   <releases>
     <release date="2023-07-17" version="4.0.4"/>
-- 
2.44.0

